Problem 1.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <numeric>
+
+using namespace std;
+
+int main()
+{
+	vector<string> a;
+	string p;
+	do {
+		getline(cin, p);
+		a.push_back(p);
+	} while (p != "");
+	a.pop_back();
+
+	if (all_of(a.begin(), a.end(), [](string i) {return (i.size() >= 4); })) {
+		cout << "ALL\n";
+	}
+	else if (any_of(a.begin(), a.end(), [](string i) {return (i.size() >= 4); })) {
+		cout << "SOME\n";
+	}
+	else if (none_of(a.begin(), a.end(), [](string i) {return (i.size() >= 4); })) {
+		cout << "NONE\n";
+	}
+
+	for_each(a.begin(), a.end(), [](string i) {cout << i.back() << "\n"; });
+
+	cout << count_if(a.begin(), a.end(), [](string i) {return count_if(i.begin(), i.end(), isdigit); }) << "\n";
+
+	transform(a.begin(), a.end(), a.begin(), [](string i) {transform(i.begin(), i.end(), i.begin(), tolower); return 0; });
```

Comment:
I don't believe this is quite right still. You are returning 0, not the new string. Print out the contents of the vector after doing this transform and see what happens!